src/styles/modules/_spacings.scss

```diff
@@ -0,0 +1,49 @@
+@use 'sass:map';
+
+// Variables
+$base-unit: 0.25rem;
```

Comment:
One doubt, normally the padding and margins will be pixels, right?

Reply:
you are right, I'll change it, but for this iteration I think it is not necessary to add that cover.